invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LocationList/components/InternalLocationList/InternalLocationList.js

```diff
@@ -24,7 +20,7 @@ export default class InternalLocationList extends Component {
   }
 
   handleOnRefClick(itemPid) {
-    openRecordEditor(itemApi.url, itemPid);
+    // edit record
```

Comment:
Or leave a comment like this one in the places you removed the `openRecordEditor()` :)

Reply:
or TODO, some editors are easily detecting it